docker/tests/Templates/fxdependent-centos7/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM centos:7
```

Comment:
We should base this off the 2.1-runtime and 2.1-sdk image flavors from https://hub.docker.com/r/microsoft/dotnet/tags/

Reply:
Updated this for ubuntu18.04 and debian.9;
Unfortunately they don't have any other images.

Reply:
Do we need to test all these platforms?